src/Std/Time/Zoned/Database/Windows.lean

```diff
@@ -0,0 +1,83 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+prelude
+import Std.Time.DateTime
+import Std.Time.Zoned.TimeZone
+import Std.Time.Zoned.ZoneRules
+import Std.Time.Zoned.Database.Basic
+
+namespace Std
+namespace Time
+namespace Database
+
+set_option linter.all true
+
+namespace Windows
+
+/--
+Fetches the next timezone transition for a given timezone identifier and timestamp.
+-/
+@[extern "lean_windows_get_next_transition"]
+opaque getNextTransition : @&String -> @&Int64 -> IO (Option (Int64 × TimeZone))
```

Comment:
`Int64` parameters shouldn't receive a borrow annotation. Also applies to the function below.